test-packages/type-tests/test/index.d.ts

```diff
@@ -1 +1,2 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+// Minimum TypeScript Version: 3.9
```

Comment:
Is this necessary if you are using your local ts version?

Reply:
I found this config useful since we are using some 3.9 features.
But you are right, our local ts is configured to be used.